ko/pl.rst

```diff
@@ -0,0 +1,1383 @@
+        | <label_declaration>? FOR <iterator> LOOP <seq_of_statements> END LOOP           # for-iter-loop
+        | <label_declaration>? FOR <for_cursor> LOOP <seq_of_statements> END LOOP         # for-cursor-loop
+        | <label_declaration>? FOR <for_static_sql> LOOP <seq_of_statements> END LOOP     # for-static-sql-loop
+        | <label_declaration>? FOR <for_dynamic_sql> LOOP <seq_of_statements> END LOOP    # for-dynamic-sql-loop
+
+    <label_declaration> ::= << <identifier> >>
+
+    <iterator> ::= <identifier> IN [ REVERSE ] <lower_bound> .. <upper_bound> [ BY <step> ]
+
+    <for_cursor>      ::= <record> IN <cursor_expression> [ <function_argument> ]
+    <for_static_sql>  ::= <record> IN ( <select_statement> )
+    <for_dynamic_sql> ::= <record> IN ( EXECUTE IMMEDIATE <dynamic_sql> [ <using_clause> ] )
+
+* *label_declaration*: 오직 루프문 시작 부분에서만 라벨 선언을 할 수 있다. 이 라벨은 루프 바디 안 쪽의 CONTINUE 문이나 EXIT 문이 분기 기준이 될 루프를 지정하는데 사용된다.
+* *for-iter-loop* 형태의 루프에서 *lower_bound*, *upper_bound*, *step*\은 모두 INTEGER 타입을 갖는다. 루프 변수 *identifier*\는 루프 바디 안에서 INTEGER 타입 변수로 사용될 수 있다.
+* *for-cursor-loop*, *for-static-sql-loop*, *for-dynamic-sql-loop* 형태의 FOR 루프는 *record* IN 다음에 기술하는 SELECT 문의 조회 결과들을 순회하기 위해 사용된다. 매 iteration 마다 조회 결과가 한 row 씩 *record*\에 할당된 상태로 루프 바디가 실행된다. 이 때, 결과 row의 각 컬럼들은 루프 바디 안에서 *record*. *column* 모양으로 참조할 수 있다.
+* *for-dynamic-sql-loop* 문 안에서의 *using_clause*\는 EXECUTE IMMEDIATE 문에서와는 달리 OUT 키워드를 지정할 수 없다.
+
+다음은 For-Iterator Loop 구문의 사용예를 보여준다.
+
+.. code-block:: sql
+
+    CREATE PROCEDURE mult_tables AS
+    BEGIN
+        FOR i IN 2 .. 9 LOOP
+            put_line('table ' || i);
+
+            FOR j IN 1 .. 9 LOOP
+                put_line(i || ' x ' || j || ' = ' || i*j);
+            END LOOP;
+
+            put_line('');
+        END LOOP;
+    END;
+
+다음은 동일한 SELECT 문을 세 가지 다른 형태의 For Loop으로 조회하는 예를 보여준다.
+
+.. code-block:: sql
+
+    CREATE PROCEDURE athlete_history(name VARCHAR) AS
+        CURSOR my_cursor IS SELECT host_year, score FROM history WHERE athlete = name;
+    BEGIN
+        -- For-Cursor Loop
+        FOR r IN my_cursor LOOP
+            put_line('host_year: ' || r.host_year || ' score: ' || r.score);
+        END LOOP;
+
+        -- For-Select Loop
+        FOR r IN (SELECT host_year, score FROM history WHERE athlete = name) LOOP
+            put_line('host_year: ' || r.host_year || ' score: ' || r.score);
+        END LOOP;
+
+        -- For-Dynamic-SQL Loop
+        FOR r IN (EXECUTE IMMEDIATE 'SELECT host_year, score FROM history WHERE athlete = ?' USING name) LOOP
+            put_line('host_year: ' || r.host_year || ' score: ' || r.score);
+        END LOOP;
+    END;
+
+.. _case_stmt:
+
+CASE
+====
+CASE 문은 여러 개의 조건을 순차적으로 검사해서 가장 처음 만족하는 조건에 연관되어 있는 실행문들을 실행한다.
+
+::
+
+    <case_statement> ::=
+          CASE <expression> { WHEN <expression> THEN <seq_of_statements> }+ [ ELSE <seq_of_statements> ] END [ CASE ]
```

Comment:
I think it would be better to write it according to "매뉴얼 소개 > 매뉴얼 규약".

```suggestion
          CASE <expression> { WHEN <expression> THEN <seq_of_statements> }... [ ELSE <seq_of_statements> ] END [ CASE ]
```